helmfile/charts/networkpolicy-service/templates/harbor-np.yaml

```diff
@@ -619,4 +630,27 @@ spec:
     - ports:
         - port: 53
           protocol: UDP
+---
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: allow-init-harbor-job
+  namespace: harbor
+spec:
+  policyTypes:
+    - Egress
+  podSelector:
+    matchLabels:
+      job-name: init-harbor-job
+  egress:
+    - to:
+        - podSelector:
+            matchLabels:
+              component: core
+      ports:
+        - port: 8080
+          protocol: TCP
+    - ports:
+        - port: 53
+          protocol: UDP
```

Comment:
Not really needed, as there is a allow dns in the namespace.

Reply:
But the init job doesn't have the same labels so it's not catched with that rule

Reply:
Yes you are right :+1: